.github/workflows/lint-python.yml

```diff
@@ -1,24 +1,29 @@
-name: Lint
+name: Lint Python
 on:
   push:
     branches-ignore:
       - "dependabot/*"
+    paths:
+      - '**/*.py'
+      - '.flake8'
+      - .github/workflows/lint-python.yml
   pull_request:
+    paths:
+      - '**/*.py'
```

Comment:
Node.js has Python files that do not have a .py suffix.

Reply:
In this repo?